internal/cli/atlas/backup/compliancepolicy/encryptionatrest/disable_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build unit
+
+package encryptionatrest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	mocks "github.com/mongodb/mongodb-atlas-cli/internal/mocks/atlas"
+	"github.com/mongodb/mongodb-atlas-cli/internal/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201004/admin"
+)
+
+func TestDisableBuilder(t *testing.T) {
+	test.CmdValidator(
+		t,
+		DisableBuilder(),
+		0,
+		[]string{
+			flag.ProjectID,
+			flag.Output,
+			flag.EnableWatch,
+		},
+	)
+}
+
+func TestDIsableOpts_InitStore(t *testing.T) {
+	opts := &DisableOpts{}
+
+	require.NoError(t, opts.initStore(context.TODO())())
+	assert.NotNil(t, opts.store)
+}
+
+func TestDisableOpts_Watcher(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockCompliancePolicyEncryptionAtRestDisabler(ctrl)
+
+	opts := &DisableOpts{
+		store: mockStore,
+	}
+
+	expected := &atlasv2.DataProtectionSettings{
+		State: atlasv2.PtrString(active),
+	}
+
+	mockStore.
+		EXPECT().
+		DescribeCompliancePolicy(opts.ProjectID).
+		Return(expected, nil).
+		Times(1)
+
+	res, err := opts.watcher()
+	require.NoError(t, err)
+	assert.True(t, res)
+}
+
+func TestDisableOpts_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockCompliancePolicyEncryptionAtRestDisabler(ctrl)
+	encryptionAtRestBefore := true
+	encryptionAtRestAfter := false
+
+	initial := &atlasv2.DataProtectionSettings{
+		EncryptionAtRestEnabled: &encryptionAtRestBefore,
+	}
```

Comment:
if you remove this nothing changes on your test `opts.policy` is only used fro the watch operation, is not really the initial value